cli/cli.go

```diff
@@ -284,6 +290,31 @@ func fetchIndex() {
 	}
 }
 
+func installFromVersionFile() {
+	blob, err := ioutil.ReadFile(VERSION_FILE)
+	if err != nil {
+		fmtc.Println("Cannot read %s", VERSION_FILE)
+		exit(1)
+	}
+
+	version, err := readFirstWord(string(blob))
+	if err != nil {
+		fmtc.Println("Cannot find version in %s", VERSION_FILE)
```

Comment:
Because, even if the error wasn't a pun, it is generic and does not provide actionable information to the user. It needs to be adapted to the actual use case.